spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -123,6 +123,13 @@
       expect(JSON.parse(response.body)['data'].count).to eq(1)
     end
 
+    it 'filters by vet_tec_provider schools' do
+      create(:institution, :vet_tec_provider, vet_tec_provider: true)
```

Comment:
Looks like `vet_tec_provider` is specified as `true` in the factory for `:vet_tec_provider`, so we don't need to specify it here:
```suggestion
      create(:institution, :vet_tec_provider)
```

Reply:
Added the change.